app/controllers/course/forum/forums_controller.rb

```diff
@@ -61,6 +61,10 @@ def unsubscribe
       flash.now[:danger] = t('.failure')
     end
     render 'update_subscribe_button'
+  rescue ActionView::MissingTemplate
```

Comment:
Does conditionally rendering on `request.delete?` or `request.get?` work? Then we can avoid raising this error in the first place.

https://guides.rubyonrails.org/action_controller_overview.html#the-request-object